.github/workflows/build-and-run-model.yaml

```diff
@@ -14,6 +14,18 @@ on:
     types: [opened, reopened, synchronize, closed]
   workflow_dispatch:
     inputs:
+      run_type:
+        type: choice
+        description: Run type or purpose
+        options:
+        - junk
+        - rejected
+        - test
+        - baseline
+        - candidate
+        - final
+        default: test
+        required: true
```

Comment:
**[Suggestion, non-blocking]** It might be useful to document the meaning of these types, either here or in the workflow run instructions in the README, but I'm fine punting on that till later.

Reply:
I have an open issue from a big write up on "how to run the model". Will save documenting these types for then.